make/include/wait_for

```diff
@@ -33,7 +33,27 @@ wait_for_job() {
     esac
   done
 
-  until [[ -n "$(kubectl get jobs --namespace "${namespace}" --selector "${selector}" -o jsonpath='{.items[?(@.status.succeeded==1)].status.conditions[?(@.type=="Complete")]}')" ]]; do
+  if [[ -z "${namespace:-}" ]]; then
+    echo "Missing namespace. '-n' must be specified." >&2
+    exit 1
+  fi
+
+  if [[ -z "${selector:-}" ]]; then
+    echo "Missing selector. '-l' must be specified." >&2
+    exit 1
+  fi
+
+  if [[ -z "${time_to_sleep:-}" ]]; then
+    time_to_sleep=3
+  fi
+
+  check_args=(
+    --namespace "${namespace}"
+    --selector "${selector}"
+    --output "jsonpath='{.items[?(@.status.succeeded==1)].status.conditions[?(@.type==\"Complete\")]}'"
```

Comment:
Huh, do you need the single quotes on the value? Doesn't `--output jsonpath={.items…` work?